scripts/generateMaintenanceTestData.sh

```diff
@@ -1,22 +1,61 @@
 #!/bin/bash
-# FIXMEMAINTENANCE: please add a couple of lines about why this script exists
+#
+# Generate test data for maintenance unit tests
+#
+
+# collections for system database
+echo "== Creating collection _system/bar =="
+curl [::]:8530/_api/collection -sd \
```

Comment:
default endpoint is 8529, is that the intention?

Reply:
just leave as is. i'll fix for next round